tests/test_rects.py

```diff
@@ -0,0 +1,231 @@
+"""Test rects extraction."""
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from collections.abc import Sequence
+    from os import PathLike
+
+    from libpdf.apiobjects import ApiObjects, Chapter, Paragraph, Rect
+
+import libpdf
+from tests.conftest import (
+    PDF_RECTS_EXTRACTION,
+)
+
+
+def find_chapter(objects: ApiObjects, chapter_name: str) -> Chapter:
+    """
+    search for given chapter in the objects.
+
+    :return: found chapter
+    """
+    chapters = objects.flattened.chapters
+    ret_chapter = None
+    assert len(chapters) > 0
+    for chapter in chapters:
+        if chapter.title == chapter_name:
+            ret_chapter = chapter
```

Comment:
add a break to the loop?

Reply:
using list comprehension with next and default to None